lib/models/rating_model.dart

```diff
@@ -0,0 +1,11 @@
+class RatingModel {
+  final String id;
+  final double rating;
+
+  RatingModel({this.id, this.rating});
+
+  Map<String, dynamic> toJson() => {
+        'id': id,
+        'rating': rating,
+      };
+}
```

Comment:
I understand that the current user logged in is being give a rating, but we want to rate other users right?
Shouldn't this service grab the ID of the User whos name is being displayed on ratings page?

Reply:
I agree with @txt020 here, I overlooked this during my testing. Can you add a way to pass in any user's id into this component?